unimol_tools/pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[build-system]
```

Comment:
Can you also delete the defunct `setup.py` files please?

Reply:
How will that affect merging in changes from the parent repo in the future? I left the structure largely unchanged because I wanted to minimize merge conflicts to resolve in the future -- if removing the setup.py files won't cause further conflicts then i will remove them.

Reply:
Hmm im not sure. Merging it in would most likely bring in that version

Reply:
Hmm ok well it is likely going to come back again but for the sake of internal clarity we can remove them and see what happens in the future

Reply:
done